sdk/src/main/scala/com/horizen/mainchain/api/RpcMainchainNodeApi.scala

```diff
@@ -48,17 +57,16 @@ class RpcMainchainNodeApi(val sidechainSettings: SidechainSettings)
 
   override def sendCertificate(certificateRequest: SendCertificateRequest): SendCertificateResponse = {
     val objectMapper = new ObjectMapper()
-    val feeParam: String = if (!certificateRequest.subtractFeeFromAmount) {
-        " false " + certificateRequest.fee
-    } else ""
+
     val response = callRpc("send_certificate "
       + encloseStringParameter(BytesUtils.toHexString(certificateRequest.sidechainId)) + " "
       + certificateRequest.epochNumber + " "
-      + encloseStringParameter(BytesUtils.toHexString(certificateRequest.endEpochBlockHash))
-      + encloseJsonParameter(objectMapper.writeValueAsString(certificateRequest.backwardTransfers))
-      + feeParam
+      + encloseStringParameter(BytesUtils.toHexString(certificateRequest.endEpochBlockHash)) + " "
+      + "\"[]\" " //encloseJsonParameter(objectMapper.writeValueAsString(certificateRequest.backwardTransfers)) + " " // TODO: fix json serialization
```

Comment:
Todo: need fix to return plain json representation of backward transfer entries (without endlines and tabs).